### PR TITLE
Networking Internet & Web Module - closes #404

### LINKED ISSUE #404: Networking Internet & Web Module
refer to basics slides in reformatted networking course

### PR DESCRIPTION
**Pull request by nickrstewarttds**

### CODE REVIEW

File: topics/networking/modules/basics/README.md [hunk 1]
```diff
@@ -0,0 +1,421 @@
...
+| Network Interface Controller (NIC )| Physically receives data from a network |
+| Host adapter | Connects a host system to other network/storage devices |
+| Gateway | Allows for data transfer between networks |
+| Network bridge | Aggregates mutliple networks together |
+| Hub | Connects multiple devices together onto a single network |
+
+There are a myriad of other physical, wireless, and hybrid network devices.
+
+### Network topology
+Topology refers to the way a network is organised, both physically and logically.
```
Comment (tvaidotas) [inline topics/networking/modules/basics/README.md]: Could you give an example what's physical what's logical?

File: topics/networking/modules/basics/README.md [hunk 2]
```diff
@@ -0,0 +1,421 @@
...
+
+Originally, all networks were wired, and therefore usually adhered to one of a few specific network topologies due to limitations on physical hardware.
```
Comment (tvaidotas) [inline topics/networking/modules/basics/README.md]: Which one was used ?

What happened that changed it, and changed it into what?

File: topics/networking/modules/basics/README.md [hunk 3]
```diff
@@ -0,0 +1,421 @@
...
+
+**Bus**
```
Comment (tvaidotas) [inline topics/networking/modules/basics/README.md]: It's hard to digest what point are you trying to get across with these schemas, could you add an explanation to each explaining it a bit, example where it was used in what scenario or something similar

File: topics/networking/modules/basics/README.md [hunk 4]
```diff
@@ -0,0 +1,421 @@
...
+```
+**Tree**
+```text
+          ┌────────[Node]────────┐
+        [Node]                 [Node]
+    ┌─────┴─────┐          ┌─────┴─────┐
+  [Node]      [Node]     [Node]      [Node]
+```
+
+### P2P
```
Comment (tvaidotas) [inline topics/networking/modules/basics/README.md]:
```suggestion
### Peer-to-peer (P2P)
```

File: topics/networking/modules/basics/README.md [hunk 5]
```diff
@@ -0,0 +1,421 @@
...
+Most **Hybrid** topologies combine several of these styles together.
```
Comment (tvaidotas) [inline topics/networking/modules/basics/README.md]:
```suggestion
Most **Hybrid** topologies combine several of these network topology styles together.
```

File: topics/networking/modules/basics/README.md [hunk 6]
```diff
@@ -0,0 +1,421 @@
...
+
+Wireless technology is heavily-hybridised by design: most Internet-enabled devices are so interconnected to each other that network topology has deprecated to the level of describing much smaller, wired, or home/office network architecture.
+
+These are **peer-to-peer** (P2P) networks: every node can exchange data with any other node, with each node acting both as a client (receiving information) and a server (sending information).
```
Comment (tvaidotas) [inline topics/networking/modules/basics/README.md]: Could give an example like smartphone speaking to a smart light bulb in the house or something

File: topics/networking/modules/basics/README.md [hunk 7]
```diff
@@ -0,0 +1,421 @@
...
+## Network Characteristics
+Networks always operate on the basis of providing seamless interconnectivity between devices.
+
+Therefore, there are a number of characteristics by which all networks must adhere to, in some capacity.
+
+The most important six ways are listed below: 
+
+| Characteristic | |
+|-|-|
+| High capacity | Can the network support significant amounts of data transfer? |
```
Comment (tvaidotas) [inline topics/networking/modules/basics/README.md]:
```suggestion
| **High capacity** | Can the network support significant amounts of data transfer? |
```

File: topics/networking/modules/basics/README.md [hunk 8]
```diff
@@ -0,0 +1,421 @@
...
+| Fast speeds | How quickly does data move between nodes? |
```
Comment (tvaidotas) [inline topics/networking/modules/basics/README.md]:
```suggestion
| **Fast speeds** | How quickly does data move between nodes? |
```

File: topics/networking/modules/basics/README.md [hunk 9]
```diff
@@ -0,0 +1,421 @@
...
+| Low latency | How long does data take to move between nodes? |
```
Comment (tvaidotas) [inline topics/networking/modules/basics/README.md]:
```suggestion
| **Low latency** | How long does data take to move between nodes? |
```

File: topics/networking/modules/basics/README.md [hunk 10]
```diff
@@ -0,0 +1,421 @@
...
+| High availability | How many nodes can supply the same data? |
```
Comment (tvaidotas) [inline topics/networking/modules/basics/README.md]:
```suggestion
| **High availability** | How many nodes can supply the same data? |
```

File: topics/networking/modules/basics/README.md [hunk 11]
```diff
@@ -0,0 +1,421 @@
...
+| Redundancy | How many backup nodes are there in the case of network failure? |
```
Comment (tvaidotas) [inline topics/networking/modules/basics/README.md]:
```suggestion
| **Redundancy** | How many backup nodes are there in the case of network failure? |
```

File: topics/networking/modules/basics/README.md [hunk 12]
```diff
@@ -0,0 +1,421 @@
...
+| Security | How safe is the network's data? Is the architecture secure from attack? |
```
Comment (tvaidotas) [inline topics/networking/modules/basics/README.md]:
```suggestion
| **Security** | How safe is the network's data? Is the architecture secure from attack? |
```

File: topics/networking/modules/basics/README.md [hunk 13]
```diff
@@ -0,0 +1,421 @@
...
+
+## Network Types
+Networks are broadly classified into two types: **LAN** (**L**ocal **A**rea **N**etwork) and **WAN** (**W**ide **A**rea **N**etworks).
```
Comment (tvaidotas) [inline topics/networking/modules/basics/README.md]:
```suggestion
Networks are broadly classified into two types: 
- **L**ocal **A**rea **N**etwork (**LAN**) 
- **W**ide **A**rea **N**etworks (**WAN**) 
```

File: topics/networking/modules/basics/README.md [hunk 14]
```diff
@@ -0,0 +1,421 @@
...
+
+The key characteristics by which we can differentiate between them are as follows:
+
+| Characteristic | LAN | WAN |
+|-|-|-|
+| Geographic distribution | Small; used for e.g. university campuses | Large; used for e.g. cities |
```
Comment (tvaidotas) [inline topics/networking/modules/basics/README.md]:
```suggestion
| **Geographic distribution** | Small; used for e.g. university campuses | Large; used for e.g. cities |
```

File: topics/networking/modules/basics/README.md [hunk 15]
```diff
@@ -0,0 +1,421 @@
...
+| Data transmission rate | Fast; gigabits | Slow; megabits |
```
Comment (tvaidotas) [inline topics/networking/modules/basics/README.md]:
```suggestion
| **Data transmission rate** | Fast; gigabits | Slow; megabits |
```

File: topics/networking/modules/basics/README.md [hunk 16]
```diff
@@ -0,0 +1,421 @@
...
+| Error rate | Lower; due to local connectivity | Higher; as a result of distribution |
```
Comment (tvaidotas) [inline topics/networking/modules/basics/README.md]:
```suggestion
| **Error rate** | Lower; due to local connectivity | Higher; as a result of distribution |
```

File: topics/networking/modules/basics/README.md [hunk 17]
```diff
@@ -0,0 +1,421 @@
...
+| Communication link | 'In-house'; e.g. coaxial cables, fibre-optics, ethernet | External; e.g. phone lines, microwave links, satellite channels |
```
Comment (tvaidotas) [inline topics/networking/modules/basics/README.md]:
```suggestion
| **Communication link** | 'In-house'; e.g. coaxial cables, fibre-optics, ethernet | External; e.g. phone lines, microwave links, satellite channels |
```

File: topics/networking/modules/basics/README.md [hunk 18]
```diff
@@ -0,0 +1,421 @@
...
+| Ownership | Single; usually a single organisation or individual | Multiple; essentially several interconnecting LANs |
```
Comment (tvaidotas) [inline topics/networking/modules/basics/README.md]:
```suggestion
| **Ownership** | Single; usually a single organisation or individual | Multiple; essentially several interconnecting LANs |
```

File: topics/networking/modules/basics/README.md [hunk 19]
```diff
@@ -0,0 +1,421 @@
...
+| Communication cost | Lower; simpler operation allows for reliability | Higher; some aspects of operation may be leased or unreliable |
```
Comment (tvaidotas) [inline topics/networking/modules/basics/README.md]:
```suggestion
| **Communication cost** | Lower; simpler operation allows for reliability | Higher; some aspects of operation may be leased or unreliable |
```

File: topics/networking/modules/basics/README.md [hunk 20]
```diff
@@ -0,0 +1,421 @@
...
+
+These differences show the difference in scale between LAN and WAN - while LAN is a small, interconnected collection of nodes usually operated by a single entity across a small area, WAN connects several LAN together to allow for wider, though less reliable, communication across multiple entities.
```
Comment (tvaidotas) [inline topics/networking/modules/basics/README.md]:
```suggestion
These differences show the difference in scale between *LAN* and *WAN* - while LAN is a small, interconnected collection of nodes usually operated by a single entity across a small area, WAN connects several LAN together to allow for wider, though less reliable, communication across multiple entities.
```

File: topics/networking/modules/basics/README.md [hunk 21]
```diff
@@ -0,0 +1,421 @@
...
+
+### Network Spatial Scope
+There are a number of other network types, according to specific circumstances.
+
+The following table gives brief explanations of each network type in practice:
+
+| Name | Acronym | Function | e.g. |
+|-|:-:|-|-|
+| Nanoscale | `nano` | Electromagnetic and molecular data transmission | medical nanotechnology |
```
Comment (tvaidotas) [inline topics/networking/modules/basics/README.md]:
```suggestion
| **Nanoscale** | `nano` | Electromagnetic and molecular data transmission | medical nanotechnology |
```

File: topics/networking/modules/basics/README.md [hunk 22]
```diff
@@ -0,0 +1,421 @@
...
+| Near-Field Communication | `NFC` | Portable dual-device communication | contactless payment |
```
Comment (tvaidotas) [inline topics/networking/modules/basics/README.md]:
```suggestion
| **Near-Field Communication** | `NFC` | Portable dual-device communication | contactless payment |
```

File: topics/networking/modules/basics/README.md [hunk 23]
```diff
@@ -0,0 +1,421 @@
...
+| Body Sensor Network | `BSN` | Wearable computing devices | smartwatches, implants |
```
Comment (tvaidotas) [inline topics/networking/modules/basics/README.md]:
```suggestion
| **Body Sensor Network** | `BSN` | Wearable computing devices | smartwatches, implants |
```

File: topics/networking/modules/basics/README.md [hunk 24]
```diff
@@ -0,0 +1,421 @@
...
+| Personal | `PAN` | Individual workspace connections | Bluetooth, wireless USB |
```
Comment (tvaidotas) [inline topics/networking/modules/basics/README.md]:
```suggestion
| **Personal** | `PAN` | Individual workspace connections | Bluetooth, wireless USB |
```

File: topics/networking/modules/basics/README.md [hunk 25]
```diff
@@ -0,0 +1,421 @@
...
+| Near-me | `NAN` | Proximity-based connections | mobile technology - masts serve multiple carriers |
```
Comment (tvaidotas) [inline topics/networking/modules/basics/README.md]:
```suggestion
| **Near-me** | `NAN` | Proximity-based connections | mobile technology - masts serve multiple carriers |
```

File: topics/networking/modules/basics/README.md [hunk 26]
```diff
@@ -0,0 +1,421 @@
...
+| (Wireless) Local | `(W)LAN` | Some institution or individual usage | Ethernet, WiFi |
```
Comment (tvaidotas) [inline topics/networking/modules/basics/README.md]:
```suggestion
| **(Wireless) Local** | `(W)LAN` | Some institution or individual usage | Ethernet, WiFi |
```

File: topics/networking/modules/basics/README.md [hunk 27]
```diff
@@ -0,0 +1,421 @@
...
+| Metropolitan | `MAN` | Interconnected LANs in a geographical area via endpoint connections | Silicon Valley |
```
Comment (tvaidotas) [inline topics/networking/modules/basics/README.md]:
```suggestion
| **Metropolitan** | `MAN` | Interconnected LANs in a geographical area via endpoint connections | Silicon Valley |
```

File: topics/networking/modules/basics/README.md [hunk 28]
```diff
@@ -0,0 +1,421 @@
...
+| Wide | `WAN` | Region-spanning network transmitting data between multiple entities | frame relays |
```
Comment (tvaidotas) [inline topics/networking/modules/basics/README.md]:
```suggestion
| **Wide** | `WAN` | Region-spanning network transmitting data between multiple entities | frame relays |
```

File: topics/networking/modules/basics/README.md [hunk 29]
```diff
@@ -0,0 +1,421 @@
...
+| Cloud | `Cloud` | Massively distributed, using mutliple data centres in multiple geographic regions as a secure connector of endpoints | GCP, AWS, Azure |
```
Comment (tvaidotas) [inline topics/networking/modules/basics/README.md]:
```suggestion
| **Cloud** | `Cloud` | Massively distributed, using mutliple data centres in multiple geographic regions as a secure connector of endpoints | GCP, AWS, Azure |
```

File: topics/networking/modules/basics/README.md [hunk 30]
```diff
@@ -0,0 +1,421 @@
...
+| Internet Area | `IAN` | Direct Cloud-based endpoint connection over IP | (conceptual) |
```
Comment (tvaidotas) [inline topics/networking/modules/basics/README.md]:
```suggestion
| **Internet Area** | `IAN` | Direct Cloud-based endpoint connection over IP | (conceptual) |
```

File: topics/networking/modules/basics/README.md [hunk 31]
```diff
@@ -0,0 +1,421 @@
...
+| Interconnected Network | `Internet` | Global system of interconnected networks | TCP/IP, World Wide Web |
```
Comment (tvaidotas) [inline topics/networking/modules/basics/README.md]:
```suggestion
| **Interconnected Network** | `Internet` | Global system of interconnected networks | TCP/IP, World Wide Web |
```

File: topics/networking/modules/basics/README.md [hunk 32]
```diff
@@ -0,0 +1,421 @@
...
+#### Encoding
+Of the ASCII character set, the characters `: / ? # [ ] @` are reserved for use as delimiters of the generic URI components.
+
+If you wanted to use any of these in the 'plaintext' areas of a URI, such as by appending an email address to a URI, they would need to be percent-encoded.
+
+For instance `%3F`may appear in certain sections of a URL for a `?`.
+
+Occasionally, `&` may also appear as `&amp` for similar reasons.
+
+## Networking Tools
```
Comment (tvaidotas) [inline topics/networking/modules/basics/README.md]: Network tools should be a separate module I think

File: topics/networking/modules/basics/README.md [hunk 33]
```diff
@@ -0,0 +1,421 @@
...
+```
+
+### P2P
+Most **Hybrid** topologies combine several of these styles together.
+
+Wireless technology is heavily-hybridised by design: most Internet-enabled devices are so interconnected to each other that network topology has deprecated to the level of describing much smaller, wired, or home/office network architecture.
+
+These are **peer-to-peer** (P2P) networks: every node can exchange data with any other node, with each node acting both as a client (receiving information) and a server (sending information).
+
+## Network Characteristics
```
Comment (tvaidotas) [inline topics/networking/modules/basics/README.md]: I would consider this a separate module too

File: topics/networking/modules/basics/README.md [hunk 34]
```diff
@@ -0,0 +1,421 @@
...
+| Characteristic | |
+|-|-|
+| High capacity | Can the network support significant amounts of data transfer? |
+| Fast speeds | How quickly does data move between nodes? |
+| Low latency | How long does data take to move between nodes? |
+| High availability | How many nodes can supply the same data? |
+| Redundancy | How many backup nodes are there in the case of network failure? |
+| Security | How safe is the network's data? Is the architecture secure from attack? |
+
+## Network Types
```
Comment (tvaidotas) [inline topics/networking/modules/basics/README.md]: I would consider this a separate module too

File: topics/networking/modules/basics/README.md (hunk 32 above)
Comment (nickrstewarttds) [reply topics/networking/modules/basics/README.md]: will split this out

File: topics/networking/modules/basics/README.md (hunk 34 above)
Comment (nickrstewarttds) [reply topics/networking/modules/basics/README.md]: will split this out

File: topics/networking/modules/basics/README.md (hunk 33 above)
Comment (nickrstewarttds) [reply topics/networking/modules/basics/README.md]: will split this out

File: topics/networking/modules/internet-and-web/README.md [hunk 35]
```diff
@@ -0,0 +1,150 @@
...
+		- [URL](#url)
+		- [Encoding](#encoding)
+- [Tasks](#tasks)
+	- [Categorise each section of a long URL](#categorise-each-section-of-a-long-url)
+	- [Explain why URL identifiers might change](#explain-why-url-identifiers-might-change)
+	- [Try sending a text to an email address](#try-sending-a-text-to-an-email-address)
+
+<!--TOC_END-->
+## Overview
+A network is essentially a web of distributed communication between entities.
```
Comment (tvaidotas) [inline topics/networking/modules/internet-and-web/README.md]:
```suggestion
A **network** is essentially a web of distributed communication between entities.
```

File: topics/networking/modules/internet-and-web/README.md [hunk 36]
```diff
@@ -0,0 +1,150 @@
...
+
+At the lowest level, networks facilitate the sharing of information.
+
+So far, we have looked at the **internal** transfer for information across a single network.
```
Comment (tvaidotas) [inline topics/networking/modules/internet-and-web/README.md]:
```suggestion
So far, previous modules covered **internal** transfer of information across a *single network*.
```

File: topics/networking/modules/internet-and-web/README.md [hunk 37]
```diff
@@ -0,0 +1,150 @@
...
+
+However, communication also happens between entirely separate networks.
```
Comment (tvaidotas) [inline topics/networking/modules/internet-and-web/README.md]:
```suggestion
However, information can be exchanged between entirely separate networks.
```

File: topics/networking/modules/internet-and-web/README.md [hunk 38]
```diff
@@ -0,0 +1,150 @@
...
+
+These networks have to connect to each other to share information, too.
```
Comment (tvaidotas) [inline topics/networking/modules/internet-and-web/README.md]:
```suggestion
There is a requirement for the networks to be connected with each other, before they can share the information.
```

File: topics/networking/modules/internet-and-web/README.md [hunk 39]
```diff
@@ -0,0 +1,150 @@
...
+
+Therefore, the **external** transfer of information is just as crucial for allowing communication between entities.
+
+## The Interconnected Network
+The most well-known network is the **Internet**, which, though well-known, is often poorly-understood.
```
Comment (tvaidotas) [inline topics/networking/modules/internet-and-web/README.md]:
```suggestion
The most well-known *network* is the **Internet**, which, though well-known, is often poorly-understood.
```

File: topics/networking/modules/internet-and-web/README.md [hunk 40]
```diff
@@ -0,0 +1,150 @@
...
+
+As the largest telecommunications network (its name is literally shortened from **inter**connected **net**work), the Internet is the best example of a WAN taken to its logical extreme.
```
Comment (tvaidotas) [inline topics/networking/modules/internet-and-web/README.md]:
```suggestion
As the largest telecommunications network *internet* (name is an abbreviation of **inter**connected **net**work), the *internet* is the best example of a WAN taken to its logical extreme.
```

Comment (tvaidotas) [reply topics/networking/modules/internet-and-web/README.md]: additionally what's WAN? don't believe it's mentioned in this module before so that the abbreviation would be known

File: topics/networking/modules/internet-and-web/README.md [hunk 41]
```diff
@@ -0,0 +1,150 @@
...
+
+As the decentralised and distributed global system of interconnected networks, it uses a series of protocols to function, most notably:
+ - **T**ransmission **C**ontrol **P**rotocol **(TCP)** 
+ - **I**nternet **P**rotocol **(IP)**
+ 
+ The combination of these results in **TCP/IP**, which serves to provide end-to-end data communication to every device that uses it.
+
+### The World Wide Web
+Contrary to popular belief, the Internet and the **World Wide Web** are not the same thing:
+ - The Internet is a *networking infrastructure* - a 'network of networks' in which any computer can communicate with another computer.
```
Comment (tvaidotas) [inline topics/networking/modules/internet-and-web/README.md]:
```suggestion
 - The **internet** is a *networking infrastructure* - a 'network of networks' in which any computer can communicate with another computer.
```

File: topics/networking/modules/internet-and-web/README.md [hunk 42]
```diff
@@ -0,0 +1,150 @@
...
+ - The Web is an *information-sharing system* which uses the Internet as its base.
```
Comment (tvaidotas) [inline topics/networking/modules/internet-and-web/README.md]:
```suggestion
 - The **WWW web** is an *information-sharing system* which uses the *internet* as its base.
```

File: topics/networking/modules/internet-and-web/README.md [hunk 43]
```diff
@@ -0,0 +1,150 @@
...
+
+Web services mostly use **H**yper**T**ext **T**ransfer **P**rotocol **(HTTP)** to transmit data.
+
+**Web Browsers** help to facilitate that data transfer, by displaying **Web pages**.
+
+Web pages are nearly always written in **H**yper**T**ext **M**arkup **L**anguage **(HTML)**.
+
+However, the Web itself is only one of many vehicles which use the Internet as its basis.
```
Comment (tvaidotas) [inline topics/networking/modules/internet-and-web/README.md]:
```suggestion
However, the Web itself is only one of many vehicles which use the *internet* as its basis.
```

File: topics/networking/modules/internet-and-web/README.md [hunk 44]
```diff
@@ -0,0 +1,150 @@
...
+
+### Web access
+Everything on the Web has its own unique address. In order to access that data, you will need that address to do it.
```
Comment (tvaidotas) [inline topics/networking/modules/internet-and-web/README.md]:
```suggestion
Everything on the *Web* has its own unique address. 

In order to access that data, you will need that address to do it.
```

File: topics/networking/modules/internet-and-web/README.md [hunk 45]
```diff
@@ -0,0 +1,150 @@
...
+
+Every web address adheres to the same standard: **U**niform **R**esource **I**ndicators **(URI)**.
```
Comment (tvaidotas) [inline topics/networking/modules/internet-and-web/README.md]:
```suggestion
Every *web address* adheres to the same standard: **U**niform **R**esource **I**ndicators **(URI)**.
```

File: topics/networking/modules/internet-and-web/README.md [hunk 46]
```diff
@@ -0,0 +1,150 @@
...
+
+Just like postal addresses, every Web resource must have a specific string of characters which unambiguously define where that particular object is.
```
Comment (tvaidotas) [inline topics/networking/modules/internet-and-web/README.md]:
```suggestion
Just like postal addresses, every *Web* resource must have a specific string of characters which unambiguously define where that particular object is.
```

File: topics/networking/modules/internet-and-web/README.md [hunk 47]
```diff
@@ -0,0 +1,150 @@
...
+
+#### URI
+URI syntax consists of a hierarchical sequence of five components:
```
Comment (tvaidotas) [inline topics/networking/modules/internet-and-web/README.md]:
```suggestion
*URI* syntax consists of a hierarchical sequence of five components:
```

File: topics/networking/modules/internet-and-web/README.md [hunk 48]
```diff
@@ -0,0 +1,150 @@
...
+```text
+  scheme :// [authority] path ? [query] # [fragment]
+```
+where the authority component divides into three subcomponents:
```
Comment (tvaidotas) [inline topics/networking/modules/internet-and-web/README.md]:
```suggestion
Where the authority component divides into three sub-components:
```

File: topics/networking/modules/internet-and-web/README.md [hunk 49]
```diff
@@ -0,0 +1,150 @@
...
+```text
+  [userinfo] @ host : [port]
+```
+Each URI begins with a **scheme**, and is followed by any **identifiers** which can be within that scheme.
```
Comment (tvaidotas) [inline topics/networking/modules/internet-and-web/README.md]:
```suggestion
Each *URI* begins with a **scheme**, and is followed by any **identifiers** which can be within that scheme.
```

File: topics/networking/modules/internet-and-web/README.md [hunk 50]
```diff
@@ -0,0 +1,150 @@
...
+```
+
+```text
+https://en.wikipedia.org/wiki/Romeo_%2B_Juliet
+```
+
+## Tasks
+
+### Categorise each section of a long URL
+For this task, navigate to [this](https://www.smile.amazon.co.uk/gp/product/B07HRHM8F3/ref=ox_sc_saved_title_1?smid=A3P5ROKL5A1OLE&psc=1) Amazon item listing.
```
Comment (tvaidotas) [inline topics/networking/modules/internet-and-web/README.md]: the link doesn't work

File: topics/networking/modules/internet-and-web/README.md [hunk 51]
```diff
@@ -0,0 +1,150 @@
...
+### Explain why URL identifiers might change 
+The URL in the previous exercise looks like this:
+```text
+https://www.smile.amazon.co.uk/gp/product/B07HRHM8F3/ref=ox_sc_saved_title_1?smid=A3P5ROKL5A1OLE&psc=1
+```
+When you open this link in your browser, some of it will change. Why do you think this is?
+
+Categorise the URL you are directed to into its constituent parts. What is each section of the URL doing? 
+
+### Try sending a text to an email address
```
Comment (tvaidotas) [inline topics/networking/modules/internet-and-web/README.md]: I assume the link that doesn't work is required for this part

File: topics/networking/modules/internet-and-web/README.md [hunk 52]
```diff
@@ -0,0 +1,152 @@
...
+
+### Categorise each section of a long URL
+For this task, navigate to [this](https://www.amazon.co.uk/gp/product/B07HRHM8F3/ref=ox_sc_saved_title_1?smid=A3P5ROKL5A1OLE&psc=1) Amazon item listing.
+
+### Explain why URL identifiers might change 
+The URL in the previous exercise looks like this:
+```text
+https://www.amazon.co.uk/gp/product/B07HRHM8F3/ref=ox_sc_saved_title_1?smid=A3P5ROKL5A1OLE&psc=1
+```
+When you open this link in your browser, some of it will change. Why do you think this is?
```
Comment (tvaidotas) [inline topics/networking/modules/internet-and-web/README.md]: provide an expandable answer

File: topics/networking/modules/internet-and-web/README.md [hunk 53]
```diff
@@ -0,0 +1,152 @@
...
+
+Categorise the URL you are directed to into its constituent parts. What is each section of the URL doing?
```
Comment (tvaidotas) [inline topics/networking/modules/internet-and-web/README.md]: provide an expandable answer

File: topics/networking/modules/internet-and-web/README.md [hunk 54]
```diff
@@ -0,0 +1,152 @@
...
+Categorise the URL you are directed to into its constituent parts. What is each section of the URL doing? 
+
+### Try sending a text to an email address
+*For this task, you will need a functioning device capable of using the* **M**ultimedia **M**essage **S**ervice **(MMS)** *to send texts.*
+
+It is possible, thanks to the way URI works, to send a text to an email address.
+
+Usually, your mobile service provider will convert your text to an email and send it for you.
+
+You'll need to set the contact to send a message to as some email, such as your personal email address.
```
Comment (tvaidotas) [inline topics/networking/modules/internet-and-web/README.md]: could you break this down into steps, like step one create a new contact, step two add your email to the contact etc.

now it's quite hard to follow

also mention that sending an MMS might cost them so be aware of the charges

File: topics/networking/modules/internet-and-web/README.md [hunk 55]
```diff
@@ -0,0 +1,222 @@
...
+
+```text
+https://en.wikipedia.org/wiki/Romeo_%2B_Juliet
+```
+
+## Tasks
+
+### Categorise each section of a long URL
+
+For this task, navigate to [this](https://www.smile.amazon.co.uk/gp/product/B07HRHM8F3/ref=ox_sc_saved_title_1?smid=A3P5ROKL5A1OLE&psc=1) Amazon item listing.
```
Comment (syed9569a) [inline topics/networking/modules/internet-and-web/README.md]: this link provided does not work?
